src/autocomplete/shared/variables.ts

```diff
@@ -0,0 +1,58 @@
+import * as c3 from 'antlr4-c3';
+import {Lexer as LexerType, ParseTree, Parser as ParserType, TokenStream} from 'antlr4ng';
+import {createParser} from './query';
+import {
+    CursorPosition,
+    GetParseTree,
+    LexerConstructor,
+    ParserConstructor,
+    SymbolTableVisitor,
+} from './autocomplete-types';
+import {getScope} from './symbol-table';
+import {computeTokenPosition} from './compute-token-position';
+
+export function getVariablesSuggestions<L extends LexerType, P extends ParserType>(
```

Comment:
let's rename this to `getVariableSuggestions`
double pluralization doesn't read well